content/en/monitors/guide/monitoring-sparse-metrics.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Monitoring Sparse Metrics
+kind: guide
+further_reading:
+- link: "/api/latest/monitors/#edit-a-monitor"
+  tag: "Documentation"
+  text: "Learn more about updating these settings via API"
+- link: "/dashboards/functions/interpolation/#default-zero"
+  tag: "Documentation"
+  text: "Learn more about interpolation"
+---
+
+## Overview
+
+When monitoring a metric that reports data infrequently, there are some tools and behaviors that should be considered to ensure a monitor's settings are appropriate for evaluating these queries as intended. 
+
+## Source of the monitor
+
+First, consider your source.
+
+### Metric-based monitor
+
+Is this a metric, change, anomaly or other metric-based monitor ? The following settings should be adjusted:
+
+* Make sure that **Do not require a full window of data for evaluation** is set
+* Is the data often delayed? Consider adjusting the monitor evaluation delay as well
+* Adjust the evaluation (average, max, min, sum) based on the expected frequency. The default evaluation is **average**, which may not be suited for sparse metrics.
+* If you are using **average**, think about adding an [interpolation function][1] like `default_zero()` to ensure the gaps in the metric are evaluated as zero.
+* If you are using arithmetic in your query, take a look at [Monitor Arithmetic and Sparse Metrics][2] for some further guidance.
+
+
+### Event-based Monitor 
+
+Is this a log, event or other event-based monitor? Look at the following:
+
+* Make sure that the "Missing data" setting corresponds to your expected monitor behavior: **Evaluate as zero**, **Show NO DATA**, **Show NO DATA and notify**, or **Show OK**
```

Comment:
To verify, I couldn't find these options `**Evaluate as zero**, **Show NO DATA**, **Show NO DATA and notify**, or **Show OK**`
```suggestion
* Under *Advanced options*, select the option for **Missing data options** that corresponds to your expected monitor behavior.
```